flashbax/buffers/item_buffer.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2023 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import jax
+from chex import PRNGKey
+
+from flashbax import utils
+from flashbax.buffers.trajectory_buffer import (
+    Experience,
+    TrajectoryBuffer,
+    TrajectoryBufferSample,
+    TrajectoryBufferState,
+    make_trajectory_buffer,
+)
+from flashbax.utils import add_dim_to_args
+
+
+def validate_sample_batch_size(sample_batch_size: int, max_length: int):
+    if sample_batch_size > max_length:
+        raise ValueError("sample_batch_size must be less than or equal to max_length")
+
+
+def validate_min_length(min_length: int, max_length: int):
+    if min_length > max_length:
+        raise ValueError("min_length used is too large for the buffer size.")
```

Comment:
In the item buffer, we are not calculating the `max_length` from the buffer_size (as is the case in the flat buffer); instead, the user specifies it directly. Hence, we just want to tell them that max should be > min.
```suggestion
        raise ValueError("min_length used cannot be larger than max_length.")
```